style.css

```diff
@@ -0,0 +1,97 @@
+html {
+  box-sizing: border-box;
+  font-family: "helvetica neue";
+  font-size: 20px;
+  font-weight: 200;
+}
+
+body {
+  margin: 0;
+}
+
+/* *, *:before, *:after {
+  box-sizing: inherit;
+} */
+
```

Comment:
Removed unused code. In this comment there is no need...

Reply:
DONE!